telegram/src/main/kotlin/me/ivmg/telegram/Updater.kt

```diff
@@ -36,4 +38,9 @@ class Updater {
             lastUpdateId = lastUpdate.updateId + 1
         }
     }
+
+    internal fun updaterStopPolling() {
```

Comment:
I think this should be called `stopPolling`.

I understand you used `updaterStopPolling` because of `updaterStartPolling`, but in that case was just because we have the public method used as a "launcher" and the private one.